### DISCUSSION TITLE
敢问大佬如何自建测速地址

### DISCUSSION THREAD

**Discussion by benzo970210** [XIU2/CloudflareSpeedTest#388] [讨论]
请问大佬，感觉最近默认测速的地址奔溃了，我换了一个测速地址能正常测速，为了避免下一次奔溃，我想问问如何自建测速地址

**Comment by XIU2** [comment]
Issues 置顶不都写了么。。。
https://github.com/XIU2/CloudflareSpeedTest/issues/168
